src/Components/Authentication/Desktop/ForgotPasswordForm.tsx

```diff
@@ -23,11 +24,16 @@ export class ForgotPasswordForm extends Component<
     error: this.props.error,
   }
 
+  onSubmit = (values: InputValues, formikBag: FormikProps<InputValues>) => {
+    Grepcaptcha(GrepcaptchaAction.ForgotSubmit)
```

Comment:
This capitalization is a bit unidiomatic. Wondering why?

Reply:
Ah yes I see now that only components use capitalization while looking closer... wasn't totally clear on the rules, at a glance most things look capitalized in this repo.

Reply:
Maybe something like 

```ts
triggerRecaptcha(RecaptchaAction.ForgotSubmit)
```

Another thing is, with TS importing and using enums in this way isn't really necessary due to the type-checker. Something like this: 

```
type RecaptchaAction = 'foo' | 'bar' | 'baz | 'bam'


function triggerRecaptcha(action: RecaptchaAction) {
  ..
}
```
As soon as the user types the opening paren of the function intellisense will be triggered, showeing the available options according to the `type`. This will simplify things. 

In normal JS this imported / exported enum pattern would work nicely! But in TS it's not necessary.

Reply:
Changed this to a string to avoid over-engineering at the moment. We have discussed the idea of firing actions in many contexts, both for validation and for analytics purposes -- for example, during purchase and bid flows. As our actions and contexts grow I think we should revisit this, I could see a use of enum's similar to Analytics being useful.

Reply:
Once we need to scale things out lets def revisit 👍